Zappy_IA_Report/DTC/Code/parser.py

```
@@ -0,0 +1,41 @@
+            port = int(port)
+            if port <= 0 or port > 65535:
+                raise ValueError
+        except (ValueError, TypeError):
+            exit_error("Error: Invalid or missing port (-p must be a number between 1 and 65535).")
+
+        if not isinstance(machine, str) or machine.strip() == "":
+            exit_error("Error: Invalid machine hostname (-h).")
+        parser = iaParser(
+            port=port,
+            name=name,
+            machine=machine
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)